src/screens/CreateNFT.js

```diff
@@ -136,6 +137,15 @@ class CreateNFT extends React.Component {
           createMelt,
         }
       );
+
+      if (this.props.useWalletService) {
+        return new hathorLib.SendTransactionWalletService(this.props.wallet, {
+          transaction,
+          outputs: transaction.outputs,
```

Comment:
Because we build the entire transaction data on token actions, so we just pass the transaction (with the output attribute set) to the `SendTransactionWalletService` class